client/js/components/statement/publicStatementModal/StatementModal.vue

```diff
@@ -164,59 +171,84 @@
         </div>
 
         <template v-if="hasPermission('field_statement_add_assignment') && hasPlanningDocuments">
-          <p
-            aria-hidden="true"
-            :class="prefixClass('c-statement__formblock-title u-mb-0_25 weight--bold inline-block')">
-            {{ Translator.trans('element.assigned') }}
-          </p>
-          <p
-            aria-hidden="true"
-            :class="prefixClass('c-statement__formblock u-ml u-mb-0_5 u-mt-0_5 inline-block')">
-            <template v-if="formData.r_element_id !== ''">
+          <fieldset>
+            <legend :class="prefixClass('c-statement__formblock-title')">
+              {{ Translator.trans('element.assigned') }}
+            </legend>
+
+            <button
+              v-if="formData.r_element_id === ''"
+              aria-labelledby="documentReference"
+              :disabled="formData.r_isNegativeReport !== '0'"
+              data-cy="statementModal:elementAssign"
+              @click="gotoTab('procedureDetailsDocumentlist')"
+              :class="prefixClass('btn--blank o-link--default text-left')">
```

Comment:
nitpic: since you changed things here anywy, can You please order alphabetically?
```suggestion
              v-if="formData.r_element_id === ''"
              aria-labelledby="documentReference"
              :class="prefixClass('btn--blank o-link--default text-left')"
               data-cy="statementModal:elementAssign"
              :disabled="formData.r_isNegativeReport !== '0'"
              @click="gotoTab('procedureDetailsDocumentlist')">
```

Reply:
done in 89e28da66a65e4cb7d279519e2cbc3f811d85a29